Packs/ThreatGrid/Integrations/ThreatGrid/ThreatGrid.py

```diff
@@ -867,13 +869,14 @@ def append_to_analysis_iocs_arrays(iocs, dbots, k):
     """
     Helper for appending analysis item to ioc an dbot arrays
     """
-    iocs.append(ioc_to_readable(k))
-    dbots.append({
-        'Vendor': 'ThreatGrid',
-        'Type': 'IOC',
-        'Indicator': k['ioc'],
-        'Score': calc_score(k['severity'])
-    })
+    if k and k.get('ioc'):
+        iocs.append(ioc_to_readable(k))
+        dbots.append({
+            'Vendor': 'ThreatGrid',
```

Comment:
the vendor should match the integration name.
please fix across the integration
```suggestion
            'Vendor': 'Threat Grid',
```